checks/fileutils.go

```diff
@@ -106,3 +130,23 @@ func CheckFilesContent(shellPathFnPattern string,
 
 	return res, nil
 }
+
+// ContainsInstructions checks if the file contains instructions or not.
+// `comment` is the string or character that indicates a comment.
+func ContainsInstructions(content []byte, comment string) bool {
```

Comment:
ContainsInstructions is slightly wrong name, since it is not contains but startsWith, maybe lineStartsWith ?
also, comment var name seems incorrect, since you are creating a generic function, so maybe rename to start or something like that ?

Reply:
Maybe `ContainsCommands()`?